### PR TITLE
Support Kubernetes v1.24

### PR DESCRIPTION
**Pull request by kmdkuk**
part of cybozu-go/neco#2114

- Support Kubernetes v1.24
- Update some depencencies
- Remove part of kubelet bootstrap settings related to dockershim removal
  - ref: https://github.com/kubernetes/kubernetes/pull/106907

Signed-off-by: kouki <kouworld0123@gmail.com>

### CODE REVIEW

**Review by zeroalphat — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,82 +5,16 @@ This project employs a versioning scheme described in [RELEASE.md](RELEASE.md#ve
...
-- Support Kubernetes 1.23 (#554)
-    - Update Kubernetes to v1.23.9
-    - Update some depencencies
-    - Use KubeschedulerConfiguration v1beta3
-    - Stop using deprecated --port option of kube-scheduler
-- Minor changes to test (#555)
+- Support Kubernetes 1.24 (#584)
+    - Update Kubernetes to v1.24.8
+    - Update some dependencies
+    - Remove part of kubelet bootstrap settings related to dockershim removal
```
Comment (zoetrope) [inline CHANGELOG.md:13]: It would be good to write the option name `--network-plugin`.

File: example/docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   cke:
     container_name: cke
-    image: quay.io/cybozu/cke:1.23.0
+    image: quay.io/cybozu/cke:1.23.5
```
Comment (zoetrope) [inline example/docker-compose.yml:5]: Will this tag be updated after release?

**Review by zoetrope — APPROVED** [review]
LGTM

File: example/docker-compose.yml (hunk 2 above)
Comment (kmdkuk) [reply example/docker-compose.yml:5]: Yes, Update here after the image is pushed.

**Pull request merged**